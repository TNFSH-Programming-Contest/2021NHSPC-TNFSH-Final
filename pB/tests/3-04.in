35 222 11
38 66 8 52 159 99
153 4 114 213 165 151
21 10 156 108 102 49
149 181 152 160 213 135
203 17 130 85 106 2
78 127 143 188 87 218
50 26 61 124 121 189
8 154 179 114 84 4
5 26 55 177 11 139
73 7 131 190 127 54
47 166 32 74 152 204
94 204 64 33 37 218
122 170 157 37 194 50
182 92 117 128 220 11
58 88 212 68 113 123
188 20 196 98 183 113
146 69 96 197 112 5
137 28 58 109 207 172
46 189 136 199 190 157
36 14 101 186 171 165
199 19 183 172 197 180
146 42 33 70 217 160
188 99 155 129 55 163
95 107 74 45 191 81
15 75 145 69 33 127
33 194 84 179 163 14
94 49 69 130 89 92
76 38 79 154 102 16
42 56 209 87 161 127
168 191 43 106 102 217
12 29 138 165 80 145
7 149 207 120 111 79
194 145 71 24 222 38
110 132 191 151 46 51
174 25 179 131 82 43
0
1 5
0
2 8 2
2 9 17
0
2 32 10
2 8 1
0
1 3
2 9 14
1 31
0
2 20 26
1 25
1 28
1 5
0
1 21
1 16
1 3
0
0
1 33
1 35
2 7 9
0
1 18
1 31
0
0
1 11
4 26 12 25 22
0
0
1 20
2 12 13
3 33 28 1
0
0
0
2 29 22
2 30 35
0
1 24
2 19 34
1 11
0
2 27 3
2 7 13
1 34
1 1
0
1 10
2 23 9
1 29
0
2 18 15
0
0
1 7
0
0
1 12
0
1 1
0
1 15
3 17 27 25
1 22
1 33
0
1 10
2 24 11
1 25
1 28
0
1 6
2 32 28
1 31
1 24
1 35
0
2 26 8
1 5
0
2 29 6
1 15
1 27
0
0
2 14 27
0
2 12 27
1 24
1 17
0
1 16
2 1 23
0
1 20
3 30 28 3
0
0
0
2 30 5
1 24
1 3
1 18
1 34
1 32
1 17
2 16 15
2 8 2
0
0
1 14
0
0
1 32
1 7
1 13
1 15
1 7
0
0
4 6 10 29 25
1 14
1 23
2 5 27
2 10 35
1 34
0
0
1 4
1 19
1 18
1 31
1 9
0
0
0
1 6
0
3 33 31 25
2 22 17
0
0
2 32 4
0
2 34 2
2 11 4
1 2
2 8 28
1 23
1 3
2 13 19
0
1 1
2 22 4
1 29
0
2 26 23
0
3 20 2 31
1 11
0
1 30
0
1 13
1 20
2 18 21
0
1 35
0
0
1 9
0
3 26 8 35
1 21
1 4
1 14
2 16 21
0
0
1 20
0
3 23 6 16
2 7 19
2 10 19
3 30 24 34
0
0
3 13 26 33
0
1 16
2 21 17
0
2 19 21
0
0
0
1 5
2 12 11
0
0
2 18 32
0
1 29
0
0
1 15
2 2 4
0
0
0
2 22 30
2 6 12
0
1 14
0
1 33
