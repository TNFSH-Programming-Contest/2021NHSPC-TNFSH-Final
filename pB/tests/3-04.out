0
0
0
0
1 9
0
1 32
1 8
0
0
0
1 31
0
0
1 25
0
0
0
0
0
1 3
0
0
0
0
0
0
0
0
0
0
0
1 26
0
0
1 20
0
1 1
0
0
0
1 29
0
0
0
1 19
1 11
0
0
1 7
0
0
0
0
0
0
0
1 15
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1 10
0
0
1 28
0
1 6
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
2 12 27
1 24
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1 34
0
0
0
0
0
0
0
0
0
0
0
1 13
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1 18
0
0
0
0
0
0
0
0
2 17 22
0
0
1 4
0
0
0
1 2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1 30
0
0
0
0
0
1 35
0
0
0
0
0
0
0
1 14
0
0
0
0
0
2 16 23
0
0
0
0
0
1 33
0
0
0
0
1 21
0
0
0
1 5
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
